tests/lib/snaps/store/test-snapd-classic-content-slot/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-snapd-classic-content-slot
```

Comment:
The snaps in store directory are the ones which are created through snapcraft.yaml, this should go in the test directory and be installed doing:
"$TESTSTOOLS"/snaps-state install-local test-snapd-classic-content-slot --classic